types/postcss-sorting/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6", "ES2018.Promise"],
```

Comment:
Is `Promise` needed here? I don't see it mentioned in the types.

Reply:
Yeah, it looks like postcss uses features that need ES2018.Promise that it didn't use in older releases

<https://github.com/DefinitelyTyped/DefinitelyTyped/actions/runs/6460426090/job/17538097415?pr=66965>